pom.xml

```diff
@@ -106,6 +106,12 @@
       <version>0.32.24</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
```

Comment:
The Apache HTTP components package made my life making HTTP requests in Java so much easier lol